doc/conf.py

```diff
@@ -208,6 +208,8 @@
     r'\.MultiBlock\.remove$',
     # Enumerations
     r'\.Plot3DFunctionEnum$',
+    # VTK methods
+    r'\.override$',
```

Comment:
I thought this would work... 😕

Reply:
@dcbr, we probably don't need this, do we?

Reply:
I think this did reduce some `override` related numpydoc warnings, just not all of them?